orbit/cmd/orbit/orbit.go

```diff
@@ -616,6 +616,31 @@ func main() {
 			Platform:       osqueryHostInfo.Platform,
 		}
 
+		// Get the hardware UUID. We use a temporary RocksDB location in order to guarantee that
+		// we're getting true UUID, not a cached UUID. See
+		// https://github.com/fleetdm/fleet/issues/17934 and
+		// https://github.com/osquery/osquery/issues/7509 for more details.
+
+		oi, err := getHostInfo(osquerydPath, filepath.Join(os.TempDir(), "tmp-db"))
```

Comment:
Double checking: `$TMPDIR`/`/tmp` are not migrated, right?

Reply:
Argh, I think I got too focused on how I was testing it (I was just copying around the /opt/orbit dir). I'm pretty sure it is. I think if I create
a random string here for the name that should work?